packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.routing.js

```diff
@@ -11,18 +11,25 @@ export default /* @ngInject */ ($stateProvider) => {
       },
     },
     redirectTo: (transition) =>
-      transition
-        .injector()
-        .getAsync('backup')
-        .then((backup) => {
-          if (backup.isInactive()) {
-            return { state: 'app.dedicatedClouds.datacenter.backup.new' };
-          }
-          if (backup.isLegacy()) {
-            return { state: 'app.dedicatedClouds.datacenter.backup.legacy' };
-          }
-          return false;
-        }),
+      Promise.all([
```

Comment:
I think you should use `$q.all` as long as we are in AngularJS context. We may not be aware of what is not correctly digested by the `$scope`